src/main/java/com/simibubi/create/content/kinetics/mechanicalArm/ArmBlockEntity.java

```diff
@@ -54,10 +54,16 @@
 
 public class ArmBlockEntity extends KineticBlockEntity implements TransformableBlockEntity {
 
+	// Server Statics
+	private static final int SearchInputsRate = 20; // the rate at which the arm will search for targets while in SEARCH_INPUTS phase
```

Comment:
```suggestion
	private static final int SEARCH_INPUTS_RATE= 20; // the rate at which the arm will search for targets while in SEARCH_INPUTS phase
```

same for the other static field